src/util/pelz_io.c

```diff
@@ -841,6 +830,100 @@ ParseResponseStatus parse_pipe_message(char **tokens, size_t num_tokens)
       return RM_PRIV_FAIL;
     }
     return RM_PRIV;
+  case CMD_LOAD_CA:
+    if (num_tokens != 4)
+    {
+      return INVALID;
+    }
+
+    if (pelz_load_file_to_enclave(tokens[3], &handle))
+    {
+      pelz_log(LOG_INFO, "Invalid extension for ca load call");
+      pelz_log(LOG_DEBUG, "Path: %s", tokens[3]);
+      return INVALID_EXT_CERT;
+    }
+
+    add_cert_to_table(eid, &ret, CA_TABLE, handle);
+    if (ret != OK)
+    {
+      pelz_log(LOG_ERR, "Add cert failure");
+      switch (ret)
+      {
+      case ERR_REALLOC:
+        pelz_log(LOG_ERR, "CA Table memory allocation greater then specified limit.");
+        break;
+      case ERR_BUF:
+        pelz_log(LOG_ERR, "Charbuf creation error.");
+        break;
+      case ERR_X509:
+        pelz_log(LOG_ERR, "X509 allocation error.");
+        return X509_FAIL;
+      case RET_FAIL:
+        pelz_log(LOG_ERR, "Failure to retrieve data from unseal table.");
+        break;
+      case NO_MATCH:
+        pelz_log(LOG_ERR, "Cert entry and CA ID lookup do not match.");
+        break;
+      case MEM_ALLOC_FAIL:
+        pelz_log(LOG_ERR, "Cert List Space Reallocation Error");
+        break;
+      default:
+        pelz_log(LOG_ERR, "ca_table_add return not defined");
+      }
+      return LOAD_CA_FAIL;
+    }
+    return LOAD_CA;
+  case CMD_REMOVE_CA:
+    if (num_tokens != 4)
+    {
+      return INVALID;
+    }
+    server_id = new_charbuf(strlen(tokens[3]));
```

Comment:
Since the variable ```server_id``` is now used as a ```cert_id``` for the Server and CA tables then we should change this variable name to ```cert_id``` in all locations.